### ISSUE TITLE
nullREAL i 980?

### ISSUE THREAD

**Issue by mittinatten** [realfagstermer/realfagstermer#521]
Er dette en effekt av en batch-rettelse?

![capture](https://cloud.githubusercontent.com/assets/2842140/16039320/6263822e-3229-11e6-840c-65e12416e438.PNG)

@knuthe @bibliomari

**Comment by mittinatten** [comment]
Her er en annen variant

![capture](https://cloud.githubusercontent.com/assets/2842140/16039426/0b446b7e-322a-11e6-8cde-84f07a04eb46.PNG)

(objekt 991433988304702201)

**Comment by danmichaelo** [comment]
Her var det tre ting:
- 980 var et eksperiment som ligger på ca. 1000 poster.  Vi må kjøre en almajobb for å slette disse igjen.
- Vi må også kjøre en jobb der vi erstatter "(No-TrBIB)" med "(NoOU)"
- Merkelig at "(NoOU)" ikke er med i `$$0` på den siste posten. Vi undersøker.

**Comment by bibliomari** [comment]
Takk, Simon! Det siste var at emneordet er autorisert i Alma. Dette må vi nok oppklare med BIBSYS, så vi får bare en variant av id-er...

**Comment by bibliomari** [comment]
Simon, vi har lagt inn et ekstra emneord i din siste post, skal se om det er Alma som selv legger til dollarnull mittinatten. Vet du om noen av dere har prøvd å autorisere med F3, i stedet for å lime inn fra ROald?

**Comment by danmichaelo** [comment]
Og så må vi finne ut om parentesen faktisk er nødvendig. Man skulle jo egentlig tro at det holdt med vokabularkode i `$$2` og at parentesen ikke var nødvendig. Men [marc-specen](https://www.loc.gov/marc/bibliographic/ecbdcntf.html) er litt uklar (den sier `"is preceded by"`, ikke `"can be preceded by"` :))

> The control number or identifier is preceded by the appropriate MARC Organization code (for a related authority record) or the Standard Identifier source code (for a standard identifier scheme), enclosed in parentheses. See MARC Code List for Organizations for a listing of organization codes and Standard Identifier Source Codes for code systems for standard identifiers. Subfield $0 is repeatable for different control numbers or identifiers.

**Comment by mittinatten** [comment]
Jeg har ikke brukt F3 (kanskje prøvd en gang) og tror ikke noen andre her heller har brukt det. Det var meg som klassifiserte begge disse.

**Comment by bibliomari** [comment]
Da frykter jeg at Alma begynner å ta kontrollen over alt.

**Comment by mittinatten** [comment]
De ble klassifisert og bestilt før jul tror jeg, hvis det kan være behjelpelig info?

**Comment by danmichaelo** [comment]
Vi har jo vært gjennom noen iterasjoner med ulike Sonja-varianter, så hvis det var før jul velger jeg å tro at det bare var en uferdig Sonja...

Men vi må finne ut en gang for alle om vi skal ha med parentesen eller ikke.

**Comment by mittinatten** [comment]
Ja, hvis det ikke var før jul så var det tidlig i januari, vil huske jeg brukte i hvert fall den ene i "lære å bestille"-seminar.

**Comment by danmichaelo** [comment]
Bare for å huske hva det var som skulle gjøres:

* [ ] Kjøre Alma-jobb for å slette alle 980-innførslene. Har sendt epost til Geir om dette.
* [ ] Kjøre Alma-jobb for å erstattee `(No-TrBIB)` med `(NoOU)`

**Comment by knuthe** [comment]
Blir alltid nervøs når jeg ser "nullREAL" i subject ... :-)

**Comment by knuthe** [comment]
Blir alltid nervøs når jeg ser "nullREAL" i subject ... :-)
